### PR TITLE
Initial renaming for GDAL3 move.

### PR DESCRIPTION
**Pull request by evetion**
Adapting to changes in:
- [x] yeesian/ArchGDAL.jl#76 (changed some functions/types)
- [x] JuliaGeo/GDAL.jl#73 (renamed everything 👍 )

At the moment running into:
```julia
PROJ: createGeodeticReferenceFrame: Cannot find proj.db```

### CODE REVIEW

**Comment by visr** [comment]
Do you see this error locally? And GDAL.jl is checked out to the right branch and built and its tests pass?

File: src/utils.jl [hunk 1]
```diff
@@ -23,7 +23,7 @@ const GMF = Dict(
 function mask_flags(flags::Int32)
     (f.first for f in GMF if (flags & f.second) == f.second)
 end
-mask_flags(band::ArchGDAL.RasterBand) = mask_flags(GDAL.C.GDALGetMaskFlags(Ptr{Nothing}(band.ptr)))
+mask_flags(band::ArchGDAL.IRasterBand) = mask_flags(GDAL.gdalgetmaskflags(Ptr{Nothing}(band.ptr)))
```
Comment (yeesian) [inline src/utils.jl]: I think `ArchGDAL.AbstractRasterBand` should work fine here?

**Comment by evetion** [comment]
> Do you see this error locally? And GDAL.jl is checked out to the right branch and built and its tests pass?

Yeah. Weird thing is, if I run a similar command in the REPL, everything works. But not when running tests. I'm going to temporarily commit my Manifest.toml and see whether the CI has the same issues.

File: src/utils.jl (hunk 1 above)
Comment (evetion) [reply src/utils.jl]: Thx! That also works.

**Pull request merged**